packages/arb-token-bridge-ui/src/components/TransferPanel/TokenRow.tsx

```diff
@@ -355,103 +413,30 @@ export function TokenRow({
-          <div className="flex w-full items-center space-x-2">
-            <span className="text-base font-medium text-gray-900">
+        <div className="flex w-full flex-col items-start gap-1 truncate">
+          <div className="flex w-full items-center gap-1">
+            <span className="text-base font-medium leading-none">
               {tokenSymbol}
             </span>
-            <span className="text-xs text-gray-500">{tokenName}</span>
-
-            {isArbitrumToken && (
-              <Tooltip content={arbitrumTokenTooltipContent}>
-                <StatusBadge variant="green">
-                  <CheckCircleIcon className="h-4 w-4" />
-                  <span className="text-xs">
-                    Official{isSmallScreen ? '' : ' ARB token'}
-                  </span>
-                </StatusBadge>
-              </Tooltip>
-            )}
-
+            <span className="text-xs text-white/70">{tokenName}</span>
+            {isArbitrumToken && <ArbitrumTokenBadge />}
             {isPotentialFakeArbitrumToken && (
               <Tooltip content="This token is different from the official Arbitrum token (ARB).">
                 <div className="box-border flex w-max flex-nowrap items-center gap-1 rounded-full border-[1px] border-gray-dark px-1 py-[2px] pr-2 text-sm">
-                  <ExclamationCircleIcon className="h-4 w-4 text-gray-dark" />
+                  <ExclamationCircleIcon className="h-3 w-3 text-gray-dark" />
                   <span className="text-xs text-gray-dark">Careful</span>
                 </div>
               </Tooltip>
             )}
           </div>
-
-          {token && (
-            <div className="flex w-full flex-col items-start space-y-1">
-              {/* TODO: anchor shouldn't be nested within a button */}
-              <div className="flex w-full justify-between">
-                {isDepositMode ? (
-                  <>
-                    {isL2NativeToken ? (
-                      <BlockExplorerTokenLink
-                        chain={childChain}
-                        address={token.address}
-                      />
-                    ) : (
-                      <BlockExplorerTokenLink
-                        chain={parentChain}
-                        address={token.address}
-                      />
-                    )}
-                  </>
-                ) : (
-                  <>
-                    {tokenHasL2Address ? (
-                      <BlockExplorerTokenLink
-                        chain={childChain}
-                        address={token.l2Address}
-                      />
-                    ) : (
-                      <span className="text-xs text-gray-900">
-                        This token hasn&apos;t been bridged to{' '}
-                        {getNetworkName(childChain.id)}.
-                      </span>
-                    )}
-                  </>
-                )}
-                {tokenIsBridgeable && tokenBalanceContent}
```

Comment:
there were two `tokenBalanceContent` and now the UI is restructured to have only one at the end